addons/account/models/account_journal.py

```diff
@@ -604,6 +604,25 @@ def _alias_prepare_alias_name(self, alias_name, name, code, jtype, company):
                 alias_name = f"{alias_name}-{company_identifier}"
         return self.env['mail.alias']._sanitize_alias_name(alias_name)
 
+    @api.model
+    def _ensure_unique_alias_creation(self, vals, company):
+        """ Check uniqueness of the alias name within the given alias domain.
+        :param vals: the values of the journal.
+        :return: a unique alias name.
+        """
+        alias_name = vals.get('alias_name')
```

Comment:
Why a get here ? Is there a case where `'alias_name'` cannot be present in `vals` ?

Reply:
Ah, indeed